### PR TITLE
V3.8 fix inspector interface

### PR DESCRIPTION
**Pull request by arsen2010**
Re: #https://github.com/cocos/3d-tasks/issues/15618

### Changelog

*

-------

### Continuous Integration

This pull request:

* [ ] needs automatic test cases check.
  > Manual trigger with `@cocos-robot run test cases` afterward.
* [ ] does not change any runtime related code or build configuration
  > If any reviewer thinks the CI checks are needed, please uncheck this option, then close and reopen the issue.

-------

### Compatibility Check

This pull request:

* [ ] changes public API, and have ensured backward compatibility with [deprecated features](https://github.com/cocos/cocos-engine/blob/v3.5.0/docs/contribution/deprecated-features.md).
* [ ] affects platform compatibility, e.g. system version, browser version, platform sdk version, platform toolchain, language version, hardware compatibility etc.
* [ ] affects file structure of the build package or build configuration which requires user project upgrade.
* [ ] **introduces breaking changes**, please list all changes, affected features and the scope of violation.

### CODE REVIEW

File: editor/inspector/contributions/node.js [hunk 1]
```diff
@@ -309,14 +309,13 @@ exports.template = /* html*/`
             </ui-section>
             <div class="after"></div>
         </ui-section>
-        <ui-prop class="postProcess" type="dump"></ui-prop>
-        <ui-prop class="fog" type="dump"></ui-prop>
-        <ui-prop class="shadows" type="dump"></ui-prop>
-        <ui-prop class="octree" type="dump"></ui-prop>
-        <ui-prop class="skin" type="dump"></ui-prop>
+        <ui-prop class="fog" type="dump" ui-section-config></ui-prop>
```
Comment (arsen2010) [inline editor/inspector/contributions/node.js:312]: ui-section-config 为了配套给 ui-prop dump 指定样式

Comment (doommm) [reply editor/inspector/contributions/node.js:312]: ui-section-config 配套的样式写在哪里？为什么这个时候不用 class 了

Comment (arsen2010) [reply editor/inspector/contributions/node.js:312]: 配套修改的 PR https://github.com/cocos/cocos-editor/pull/1930

Comment (arsen2010) [reply editor/inspector/contributions/node.js:312]: 需要 js 里创建 ui-section 的时候给它赋值

**Review by dogodo-cc — APPROVED** [review]

File: editor/inspector/contributions/node.js (hunk 1 above)
Comment (doommm) [reply editor/inspector/contributions/node.js:312]: 这个是 ui-prop 通用的吗？还是只针对 type=object 这一种情况？
因为实际产生作用的地方不在 ui-prop 里面，会加重后续维护起来的理解负担

**Review by VisualSJ — APPROVED** [review]

**Pull request merged**